diesel/src/sqlite/connection/mod.rs

```diff
@@ -107,6 +107,10 @@ impl Connection for SqliteConnection {
     fn get_transaction_depth(&self) -> i32 {
         self.transaction_depth.get()
     }
+
+    fn setup_helper_functions(&self) {
+        // this will be implemented at least when timestamps are supported in SQLite
```

Comment:
Implementing in `Connection` will have to happen eventually, so I figured I might as well add the skeleton for it. I left out the `unimplemented!()` just because it would crash whenever someone wanted to setup their database in SQLite. Would you rather I make a temporary function in `migrations` that's called only when the `postgres` feature is applied?